packages/yoga/src/Result/native/ExclusiveBadge.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { WellhubIcon } from '@gympass/yoga-icons';
+
+import Box from '../../Box';
+import Icon from '../../Icon';
+
+const ExclusiveBadge = () => (
+  <Box
+    ml="xxxsmall"
+    bg="neon"
+    justifyContent="center"
+    alignItems="center"
+    borderRadius="circle"
+    w="small"
+    h="small"
+  >
+    <Icon as={WellhubIcon} size={10.67} fill="text.primary" />
```

Comment:
💅 We should avoid magic numbers on props. How about create a constant for this to give it more readability?
```typescript
const BADGE_SIZE = 10.67;
```

```suggestion
    <Icon as={WellhubIcon} size={BADGE_SIZE} fill="text.primary" />
```